engines/wintermute/base/base_keyboard_state.cpp

```diff
@@ -342,8 +375,10 @@ uint32 BaseKeyboardState::keyCodeToVKey(Common::Event *event) {
 	case Common::KEYCODE_F12:
 		return kVkF12;
 	default:
-		warning("Key not handled: %d '%c'", event->kbd.keycode, event->kbd.keycode);
-		return event->kbd.keycode;
+		if (!((event->kbd.flags & Common::KBD_NON_STICKY) != 0)) {
+			warning("Key not handled: %d '%c'", event->kbd.keycode, event->kbd.keycode);
```

Comment:
The warning isn't really super precise (not that it was precise earlier either), since the key DOES return a value, just not necessarily the one you want.